cli.js

```diff
@@ -153,6 +157,27 @@ if (files.length === 0) {
 	];
 }
 
+var timeout;
+var timer;
+
+function restartTimer() {
+	clearTimeout(timer);
+
+	timer = setTimeout(onTimeout, timeout);
+}
+
+function onTimeout() {
+	logger.finish();
+	console.log('  ' + colors.error(figures.cross) + ' Exited because no new tests completed within last ' + cli.flags.timeout + ' of inactivity.');
```

Comment:
`ms` can't be added, because https://github.com/sindresorhus/ava/pull/654/files#diff-0730bb7c2e8f9ea2438b52e419dd86c9R689